nwbwidgets/misc.py

```diff
@@ -425,82 +425,92 @@ def update(
         matplotlib.Figure
 
         """
-
-        data = align_by_time_intervals(
-            self.units,
-            index,
-            self.trials,
-            start_label,
-            start_label,
-            start,
-            end,
-            order,
-            progress_bar=progress_bar,
-        )
-
-        fig, axs = plt.subplots(2, 1, figsize=figsize)
-
-        show_psth_raster(
-            data,
-            start,
-            end,
-            group_inds,
-            labels,
-            ax=axs[0],
-            progress_bar=progress_bar,
-        )
-
-        axs[0].set_title(f"PSTH for unit {self.unit_ids[index]}")
-        axs[0].set_xticks([])
-        axs[0].set_xlabel("")
-
-        if plot_type == "gaussian":
-            self.bins_ft.layout.visibility = "hidden"
-            self.bins_ft.layout.height = "0px"
-            self.gaussian_sd_ft.layout.visibility = None
-            self.gaussian_sd_ft.layout.height = None
-            # expanded data so that gaussian smoother uses larger window than is viewed
-            expanded_data = align_by_time_intervals(
-                units=self.units,
-                index=index,
-                intervals=self.trials,
-                start_label=start_label,
-                stop_label=start_label,
-                start=start - sigma_in_secs * 4,
-                end=end + sigma_in_secs * 4,
-                rows_select=order,
+        fig, axs = plt.subplots(2, len(start_labels), figsize=figsize)
```

Comment:
```suggestion
        fig, axs = plt.subplots(2, len(start_labels), figsize=figsize, shsarex=True, sharey=True)
```

Reply:
The plots look great!  I think sharex and sharey would make this look a bit tighter. Could you also adjust it so that the y axis labels only show up on the first plot?